README.md

```diff
@@ -27,16 +27,21 @@ still learning and exploring what's possible.
 * Compile errors with the Scala Presentation Compiler (`textDocument/publishDiagnostics`):
   - [ ] On build compile
   - [x] As you type
+* Integration with the sbt server:
+  - [x] Run custom sbt command on save
+  - [x] Receive compilation errors (`textDocument/publishDiagnostics`)
 * Linting with Scalafix (`textDocument/publishDiagnostics`):
   - [x] On build compile
   - [x] As you type
 * Refactoring with Scalafix:
   - [ ] Quick-fix inspections (`textDocument/codeAction`)
-  - [ ] Rename local symbol (`textDocument/rename`)
+    + [x] Remove unused imports
+  - [x] Rename local symbol (`textDocument/rename`)
```

Comment:
strictly speaking we don't use scalafix for rename because it's such a simple text edit